.github/workflows/benchmark.yml

```diff
@@ -17,78 +17,19 @@ concurrency:
 
 jobs:
   benchmark:
-    runs-on: "${{ github.repository == 'dagger/dagger' && 'dagger-g2-v0-12-5-4c' || 'ubuntu-latest' }}"
+    runs-on: "dagger-g2-main-4c"
     timeout-minutes: 10
 
     strategy:
       matrix:
         sdk: [go, python, typescript] # The SDKs you want to test
 
     steps:
-      - uses: actions/checkout@v4
-
-      - name: Install go
-        uses: actions/setup-go@v5
-        if: inputs.dev-engine == 'true'
-        with:
-          go-version: "1.22"
-          cache-dependency-path: "dev/go.sum"
-
-      - name: Install dagger
-        shell: bash
-        env:
-          DAGGER_VERSION: "v0.12.5"
-        run: |
-          if [[ -x "$(command -v dagger)" ]]; then
-            echo "::group::Checking dagger"
-            version="$(dagger --silent version | cut --fields 2 --delimiter ' ')"
-            if [[ "$version" != "$DAGGER_VERSION" ]]; then
-              echo "dagger ${version} is installed, but needed ${DAGGER_VERSION}"
-              exit 1
-            fi
-            echo "::endgroup::"
-          else
-            echo "::group::Installing dagger"
-            curl -fsSL https://dl.dagger.io/dagger/install.sh | BIN_DIR=/usr/local/bin/ sudo -E sh
-            echo "::endgroup::"
-          fi
-
-      - name: Build dev dagger
-        shell: bash
-        run: |
-          echo "::group::Starting dev engine"
-          if ! [[ -x "$(command -v docker)" ]]; then
-            echo "docker is not installed"
-            exit 1
-          fi
-
-          # put env variables in /tmp/local-envs instead of
-          # $GITHUB_ENV to avoid leaking into parent workflow
-          (cd .dagger/mage; go run main.go -w ../.. engine:dev) | tee /tmp/local-envs
-
-          echo "::endgroup::"
-
-      - name: Wait for dagger to be ready
-        shell: bash
-        run: |
-          source /tmp/local-envs
-          mkdir -p /tmp/benchmark
-          cd /tmp/benchmark
-
-          echo "::group::Dagger client version"
-          dagger version
-          echo "::endgroup::"
-
-          echo "::group::Dagger server version"
-          echo "{version}" | dagger query
-          echo "::endgroup::"
-        env:
-          DAGGER_CLOUD_TOKEN: "dag_dagger_sBIv6DsjNerWvTqt2bSFeigBUqWxp9bhh3ONSSgeFnw"
-
       - name: dagger init (${{ matrix.sdk }})
         shell: bash
         run: |
           source /tmp/local-envs
```

Comment:
I don't think that we need to `source /tmp/local-envs` - the file no longer exists.